core/filters/test/unit/price.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { price } from '@vue-storefront/core/filters/price'
+
+jest.mock('@vue-storefront/core/lib/multistore', () => ({
+  currentStoreView: jest.fn().mockReturnValue({
+    i18n: {
+      defaultLocale: 'en-US',
+      currencySign: '$',
+      currencyDecimal: '',
+      currencyGroup: '',
+      fractionDigits: 2,
+      priceFormat: '{sign}{amount}'
+    }
+  })
+}))
+
+describe('price', () => {
+  it('Check if argument is not a number', () => {
+    expect(price(undefined)).toBe(undefined)
+  })
+
+  it('Check if number is converted to string with sign', () => {
```

Comment:
Treat these comments like sentences in 3rd person.
`Check if number... -> it('checks if number...') -> which we read as 'It checkS if number...'`